static/src/stylesheets/module/content/_media-player.scss

```diff
@@ -1095,7 +1165,7 @@ $ima-controls-height: 70px;
 // we had major refactoring issues.
 .gu-media--show-controls-at-start.vjs-paused {
     .vjs-control-bar {
-        bottom: 0;
+        bottom: -3.95em;
```

Comment:
this feels a little magical - should we add a comment against the value of this property?